ovos_color_parser/models.py

```
@@ -27,6 +27,10 @@ def __hash__(self):
     def as_spectral_color(self) -> 'SpectralColor':
         return self.as_hsv.as_spectral_color
 
+    @property
+    def as_rgb(self) -> 'RGBa Tuple':
```

Ruff (0.14.8)

Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)